mantidproject/fitbenchmarking
# Fixed NIST Visual Display pages

#### Description of Work

Fixes #82 
Visual display pages are now produced for each starting position and the links are correct in the main tables.

#### Testing Instructions

1. Run `example_runScript.py` in the mantidpython console.
2. Check the links in the NIST tables and make sure that they link to the correct pages.
